apps/remix-ide/src/app/tabs/analysis-tab.js

```diff
@@ -1,9 +1,11 @@
+import React from 'react' // eslint-disable-line
 import { ViewPlugin } from '@remixproject/engine-web'
+import ReactDOM from 'react-dom'
 import { EventEmitter } from 'events'
+import {RemixUiStaticAnalyser} from '@remix-ui/static-analyser' // eslint-disable-line
 import * as packageJson from '../../../../../package.json'
+var Renderer = require('../ui/renderer')
```

Comment:
Is `Renderer` still needed?

Reply:
In some we still need the render, because once the section is commented out or removed the plugin throws an error, i am yet to understand the cause of this behavior.